packages/amplify_auth_cognito/example/pubspec.yaml

```diff
@@ -7,6 +7,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 
 environment:
   sdk: ">=2.7.0 <3.0.0"
+  flutter: ">=1.17.0 <2.0.0"
```

Comment:
Minor nit: If we are adding this now, let's make it 1.20.0 instead of 1.17.0 in order to match our docs (for this example app, anyway).

Reply:
hmm that line shouldn't be necessary.  I think it would be better if I just removed it - what do you think?

My changes should primarily be about the dart SDK version moving from >=2.7.0 to >=2.12.0